library/src/main/java/com/getstream/sdk/chat/adapter/BaseAttachmentViewHolder.java

```diff
@@ -53,9 +53,6 @@ public void setBubbleHelper(MessageListView.BubbleHelper bubbleHelper) {
         this.bubbleHelper = bubbleHelper;
     }
 
-    public MessageListViewStyle getStyle() {
-        return style;
-    }
 
```

Comment:
can you explain why you're making this change? 

@adrian09h

Reply:
All `AttachmentViewHolder`s  are subclussing from `BaseAttachmentViewHolder`

I have changed private to protected from  `BaseAttachmentViewHolder`.

```
    protected Context context;
    protected Message message;
    protected Attachment attachment;
    protected MessageListViewStyle style;
```